src/atoms/Element/Readme.md

```diff
@@ -0,0 +1,94 @@
+## What is an Element?
+An atomic element is a component which gives you access to standardized color, spacing, typography and font-weight as a component property. This allows you to compose new components with semantic HTML while adhering to the standard.
+
+An atomic element will reset it's styled margins, font-size and color.
+
+```css
+  .element {
+    margin: 0;
+    font-size: 100%;
+    vertical-align: baseline;
+    color: color('neutral-2');
+  }
+```
+
+### Element example:
+
+```example
+  <div>
+    <Element tag='h1' color='accent-3' typography={3}>h1 tags here.</Element>
+    <Element tag='p' color='neutral-4' spacer={12} typography={7}>custom p tag.</Element>
+
+    <Element tag='h3' typography={5} weight='light'>There was a
+    <Element tag='u'>spacer</Element> set above me</Element>
+  </div>
+```
+
+```js
+  // If you use the Elemental module, you can write familiar semantic Html.
+  // The above example would be written as:
+  <div>
+    <H1 color='accent-3' typography={3}>h1 tags here.</H1>
+    <P color='neutral-4' spacer={12} typography={7}>custom p tag.</P>
```

Comment:
@patrickkim Do you think you could explain `spacer` a bit better? Especially the fact that it is only bottom spacing?